src/MassTransit.RabbitMqTransport/IExchangeTypeDeterminer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public interface IExchangeTypeDeterminer
```

Comment:
I don't feel like Determiner is the best name here. Do you have examples of using that term for a type elsewhere?